data_dir/c-s_lwt_basic.yaml

```diff
@@ -6,7 +6,7 @@ keyspace: cqlstress_lwt_example
 
 # The CQL for creating a keyspace (optional if it already exists)
 keyspace_definition: |
-  CREATE KEYSPACE IF NOT EXISTS cqlstress_lwt_example WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};
+  CREATE KEYSPACE IF NOT EXISTS cqlstress_lwt_example WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 3};
```

Comment:
all those are working ? i.e. it's not just a search and replace, you nee to specific the region

Reply:
according to docs, we can do such a trick: https://docs.scylladb.com/stable/cql/ddl.html#networktopologystrategy
I'll run some jobs to verify it.

Reply:
Verified in job: https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/lukasz/job/reproducer-counters-multidc-raft/14/
with this setting it created keyspaces with actual DC names:
```
cqlsh> DESCRIBE scylla_bench

CREATE KEYSPACE scylla_bench WITH replication = {'class': 'NetworkTopologyStrategy', 'eu-westscylla_node_west': '3', 'us-eastscylla_node_east': '3', 'us-west-2scylla_node_west': '3'}  AND durable_writes = true;

cqlsh> describe keyspace1

CREATE KEYSPACE keyspace1 WITH replication = {'class': 'NetworkTopologyStrategy', 'eu-westscylla_node_west': '3', 'us-eastscylla_node_east': '3', 'us-west-2scylla_node_west': '3'}  AND durable_writes = true;
```

Reply:
o.k. didn't know that feature, is it scylla unique ? since other docs of cassandra doesn't mention it.

Reply:
me either, but then https://github.com/scylladb/scylladb/pull/13990 showed.
It looks it's scylla specific.

Reply:
But who gave you that hint? 😄

Reply:
you were just a messenger of Yaniv who found it from Jan who created this PR :stuck_out_tongue:

Reply:
The `scylla source code` disagree:

```
https://github.com/scylladb/scylladb/blob/996eac9569fba1b08ae15e2533a158fd0d23b214/locator/network_topology_strategy.cc#L51-L55
```

^ It is from the `5.2` branch

Reply:
it was introduced in 3.0: https://github.com/scylladb/scylladb/issues/4210

Reply:
Ok, looks like that part of code is not reached in this case.
No concerns then.